Dockerfile

```diff
@@ -8,24 +8,25 @@ COPY go.sum go.sum
 COPY vendor/ vendor/
 
 # Copy the go source
+COPY Makefile Makefile
 COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
 COPY internal/ internal/
 
-COPY hack/csv-generator.go csv-generator.go
+COPY hack/csv-generator.go hack/csv-generator.go
 
 # Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on make manager
```

Comment:
Maybe the docker argument needs to be passed here as en env variable to make? I haven't checked it, so I'm not sure.

Reply:
It is enough that the env vars are consumed with `ARG`, so it should be good now

Reply:
Ok